apps/optimizely/src/EditorPage/subcomponents/references-section.jsx

```diff
@@ -100,7 +100,3 @@ ReferencesSection.propTypes = {
   references: PropTypes.array,
   sdk: PropTypes.object.isRequired,
 };
-
-ReferencesSection.defaultProps = {
```

Comment:
Getting warnings in the console about this being deprecated and to just define the default when passing in props instead, so for some components I removed this usage, and then destructured the props if they weren't already to be able to define the default that way.